src/Engine/Manager/RenderManager.cpp

```diff
@@ -443,14 +457,14 @@ Component::SpotLight* RenderManager::CreateSpotLight() {
 
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The function 'GetSpotLights' is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)